react-native-gutenberg-bridge/index.js

```diff
@@ -101,8 +101,11 @@ export function getOtherMediaOptions( filter, callback ) {
 	return RNReactNativeGutenbergBridge.getOtherMediaOptions( filter, callback );
 }
 
-export function requestImageFullscreenPreview( mediaUrl ) {
-	return RNReactNativeGutenbergBridge.requestImageFullscreenPreview( mediaUrl );
+export function requestImageFullscreenPreview( currentImageUrl, originalImageUrl ) {
+	if ( isIOS ) {
+		return RNReactNativeGutenbergBridge.requestImageFullscreenPreview( currentImageUrl, originalImageUrl );
+	}
+	return RNReactNativeGutenbergBridge.requestImageFullscreenPreview( originalImageUrl || currentImageUrl );
```

Comment:
tested, fix looks good on Android as well 👍. I was initially surprised this change didn't require android changes as well, but this works great.

Reply:
On android works by sending the big image url even if we are showing a smaller version. No native changes needed.

We wouldn't need changes on iOS either, but these changes aim to help on the image editing work.

On the future, we will need these changes on Android too, since the plan is to implement image editing on Android too, but still it would be cool to use the same API on Android soon. 👍